apps/stats/src/views/Stats/Growth.tsx

```
@@ -9,228 +9,95 @@
+                label: 'Free members'
+            }));
+        case 'paid-members':
+            return allChartData.map(item => ({
+                ...item,
+                value: item.paid,
+                formattedValue: formatNumber(item.paid),
+                label: 'Paid members'
+            }));
+        case 'mrr':
+            // For MRR we'd need actual MRR data, but we can simulate it based on paid members
+            const avgMrrPerMember = 958 / paidMembers; // Estimate based on total MRR / total paid members
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)